src/panels/config/dashboard/ha-config-updates.ts

```diff
@@ -125,6 +125,22 @@ class HaConfigUpdates extends LitElement {
           height: 24px;
           width: 24px;
         }
+        button.show-more {
```

Comment:
Use <https://github.com/home-assistant/frontend/blob/dev/src/resources/styles.ts#L151-L162>

Reply:
Then it will no longer match?

Reply:
Oh that was used? <https://github.com/home-assistant/frontend/pull/11435/files#diff-ae93cc12a13745395838539659cd787e05014b8f4e6fbc262472e7f792e82bceR97>

Reply:
Then remove that 😛

Reply:
it was

Reply:
yeah, did not see the import, will remove shortly